app/src/androidTest/java/de/westnordost/streetcomplete/quests/oneway_suspects/TrafficFlowSegmentsApiTest.kt

```diff
@@ -27,7 +28,7 @@ class TrafficFlowSegmentsApiTest {
             1L to listOf(TrafficFlowSegment(LatLon(2.0, 1.0), LatLon(6.0, 5.0))),
             2L to listOf(TrafficFlowSegment(LatLon(4.0, 3.0), LatLon(8.0, 7.0)))
         )
-        assertThat(result).containsAllEntriesOf(expected)
+        assertMapsAreEqual(expected, result)
```

Comment:
Actually, shouldn't `assertEquals(expected, result)` work?

(`TrafficFlowSegment` and `LatLon` are both data classes, the rest being basic types)